README.md

```diff
@@ -144,4 +144,9 @@ for more details:
 **[statically.io](https://statically.io)** (<kbd>demo</kbd> and <kbd>examples</kbd> live environments)
 
 
+<a name="code_coverage">*️⃣ </a> The code coverage doesn't count the code that is only tested through HTML page, like MxGraph. \
+As **puppeteer-to-istanbul** library doesn't support source map, the code coverage based on e2e is on break. \
```

Comment:
you mean 'is broken'?

Reply:
According to https://www.deepl.com, _it's on break_ exists ^^ 
I don't know what it's the best.